rollup.config.js

```diff
@@ -1,44 +1,44 @@
-import typescript from 'rollup-plugin-typescript2';
-import { terser } from 'rollup-plugin-terser';
+import terser from "@rollup/plugin-terser";
```

Comment:
Can we not change `'` to `"`

Reply:
Sure, we can change it, if u like. I just used my Prettier in VSCode to do some code styling. Maybe we should create a prettier conf to keep the styling streamlined.